src/utils/tooltipUtils.ts

```diff
@@ -0,0 +1,44 @@
+import { PathwayType, Sector } from "../types";
+
+export const pathwayTypeTooltips: Record<PathwayType, string> = {
+  "Direct Policy": "Direct Policy test tooltip TKTK",
+  "Exploratory": "Exploratory test tooltip TKTK",
+  "Normative": "Normative test tooltip TKTK",
+  "Predictive": "Projection test tooltip TKTK",
```

Comment:
@jacobvjk @jdhoffa Sorry, touched the file and un-attached the comment. These are the names that match schema.